content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -189,6 +189,15 @@ spec:
 
 This ensures that even pods that aren't selected by any other NetworkPolicy will not be allowed ingress or egress traffic.
 
+## SCTP support
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Kubernetes supports SCTP as a `protocol` value in `NetworkPolicy` definitions as an alpha feature. To enable this feature, the cluster administrator needs to enable the `SCTPSupport` feature gate on the apiserver, for example, `“--feature-gates=SCTPSupport=true,...”`. When the feature gate is enabled, users can set the `protocol` field of a `NetworkPolicy` to `SCTP`. Kubernetes sets up the network accordingly for the SCTP associations, just like it does for TCP connections.
+
+The CNI plugin has to support SCTP as `protocol` value in `NetworkPolicy`.
```

Comment:
NetworkPolicy != CNI.  I would just remove this line

Reply:
For some reasons it got merged after the approve from @zparnold. I update the doc with a new issue according to this comment.

Reply:
@thockin, on the other hand, the existing documentation of Network Policy tells, that `Network policies are implemented by the network plugin, so you must be using a networking solution which supports NetworkPolicy`
So, that part of the documentation requires the support of Network Policy from the network plugin.